extensions/notebook/package.json

```diff
@@ -128,7 +128,8 @@
 		"menus": {
 			"commandPalette": [
 				{
-					"command": "notebook.command.analyzeNotebook"
+					"command": "notebook.command.analyzeNotebook",
```

Comment:
Do you plan on turning this back on at some point? If not shouldn't you just remove the entry all together?

Reply:
By default all commands show in the command palette. You need to specifically exclude them by using the `when:false` syntax to stop them showing.

In this case it works from right-click menu. If we had some smart context key setting to detect that the currently selected node in OE is a .csv or .tsv file we could re-enable, but I don't see that happening for a while.